tests/stringIndices_test.js

```diff
@@ -0,0 +1,17 @@
+import { expect } from 'chai'
+import { stringIndices } from '../src/stringIndices.js'
+
+describe('stringIndices()', () => {
+  it('should be a function', () => {
+    expect(stringIndices).to.be.a('function')
+  })
+  it('returns value of index', () => {
+    expect(stringIndices('Zeze loves dim sum', 3)).to.equal('sum')
+  })
+  it('returns empty string if calling on index that is greater than randomString.length', () => {
+    expect(stringIndices('Moar noodz por favor!', 7)).to.equal('')
+  })
+  it('returns empty string if calling on negative index', () => {
+    expect(stringIndices('Yaddadadadamean?', -2)).to.equal('')
+  })
+})
```

Comment:
In general, your tests should include more edge cases and be more robust. An interface of some type may have helped point this out.